src/components/depoimentos-section/testemonials-carousel/slide/slide.tsx

```diff
@@ -0,0 +1,27 @@
+import styles from "../styles.module.css";
+
+type SlideProps = {
+  quote: string;
+  content: string;
+  author: string;
+};
+
+export function Slide({ quote, content, author }: SlideProps) {
+  return (
+    <li>
+      <article className={styles.slide}>
+        <div className={styles.card_header}>
+          <div className={styles.card_header_quote}>{quote}</div>
+        </div>
+        <div className={styles.card_content}>
+          {content.split("\n\n").map((paragrafo, index) => (
+            <>
```

Comment:
precisa desse fragmento aqui?

Reply:
ah, outra coisa, quando a gente usa um map, o 1º elemento precisa ter a key, nesse caso a key tá no `<p>` não no `<>`, aí o React não identifica essa key, então quando for necessário usar um fragmento (que acho que aqui não é), tu pode usar a outra sintaxe do fragmento e passar a key, ficaria mais ou menos assim:

```
import { Fragment } from "react";

export function Component(){
  return (
   { data.map(item => (
     <Fragment key={item.id}>
       <p>Hello</p>
       <p>World</p>
     </Fragment>
   ))}
  )
}
```

Reply:
Retirei o fragmento então a key está no `<p>`, esse é o commit [32be67e](https://github.com/tech-pro-bem/abre/pull/30/commits/32be67e5575f2e28a4b9d42fe7d9eee83d51dd09)